sqlmesh/utils/pydantic.py

```diff
@@ -262,18 +265,83 @@ def positive_int_validator(v: t.Any) -> int:
     return v
 
 
+def _get_expressions(
+    v: t.Any,
+    values: t.Union[t.Dict, ValidationInfo],
+    coerce_func: t.Optional[t.Callable[[t.Any], t.Any]] = None,
+) -> t.List[exp.Expression]:
+    coerce_func = coerce_func or (lambda x: x)
+    values = values if isinstance(values, dict) else values.data
+    dialect = values.get("dialect")
+
+    if isinstance(v, (exp.Tuple, exp.Array)):
+        expressions: t.List[exp.Expression] = v.expressions
+    elif isinstance(v, exp.Expression):
+        expressions = [v]
+    else:
+        expressions = [
+            parse_one(entry, dialect=dialect) if isinstance(entry, str) else entry
+            for entry in ensure_list(v)
+        ]
+
+    results = []
+
+    for expr in expressions:
+        expr = normalize_identifiers(
+            exp.column(expr) if isinstance(expr, exp.Identifier) else coerce_func(expr),
```

Comment:
why did we add this all of a sudden?

Reply:
Working on splitting out validators for creating Expressions and Columns

Reply:
i don't believe this is correct, we may not always want columns in any generic list of expressions